manifests/params.pp

```diff
@@ -62,6 +68,14 @@
   $email_from = "no-reply@${::domain}"
   $email_enabled = false
 
+  $lazy_enabled = undef
+  $lazy_redirect_host = undef
+  $lazy_redirect_port = undef
+  $lazy_redirect_path = undef
+  $lazy_https_retrieval = undef
+  $lazy_download_interval = undef
+  $lazy_download_concurrency = undef
```

Comment:
I think that is only appropriate if the 'default' is '' or nil
if there is a default value (i.e. false) lets juts use that